### DISCUSSION TITLE
Batch create not work

### DISCUSSION THREAD

**Discussion by breakingbad45** [pocketbase/pocketbase#5941] [Q&A] [answered]
```
import pb from '../util/pocketbase';
const batch = pb.createBatch();
const bulkEntry =()=>{

cosnt newArray= [{...},{....}]
 try {
        await batch.collection('catalog_duplicate').create(newArray);   
      console.log("Data successfully added!");
    } catch (error) {
      console.error("Error creating entries:", error);
    }
}
```

**Comment by ganigeorgiev** [comment, marked as answer]
I don't know from where did you copy it but the above code sample doesn't make sense (you are not even sending the batch request). You can explore the sample code generated in the API preview for your collection.

Here is an example:

```js
const batch = pb.createBatch();

for (let item of newArray) {
    batch.collection('messages').create(item);
}

const result = await batch.send();
```


In addition, make sure that:
- the Batch API is enabled from the _Dashboard > Settings > Batch API_ panel.
- your client request executes within the configured batch limits
- your client request satisfies the Create API rule of your collection

You can also start the executable with `--dev` and/or check the logs in the _Dashboard > Logs_ section for more detailed errors.

_As a side-note, I've asked you in the past to not treat the discussions and issues tracker as AI chat but you refuse to do so (I also suspect that @apujayed is your other account). Please try to provide more details with your questions, otherwise it is just lazy and low-effort._
